packages/word_counter/src/index.ts

```diff
@@ -0,0 +1,14 @@
+// In Webpack 4 and earlier, synchronous loading of WASM isn't supported.
+// So instead, we do an old-style async import(), and export the promise.
+// It's up to the caller to resolve it at some point.
+let word_counter_promise = import('./lib/word_counter');
+
+/** Counts the number of words in the given JSON blob full of Quill deltas.
+* @param {string} bodyText A JSON blob made of Quill Deltas.
+* @returns {number} The number of words in the submitted text.
+*/
+export async function countWords(bodyText: string): Promise<number> {
+    let wasm_module = await word_counter_promise;
+    return wasm_module.count_words(bodyText);
+}
```

Comment:
This is the "minimal boilerplate" I meant. Basically, we have to wrap the WASM module up in a little bit of asncy loading because Webpack 4 _cannot_ synchronously load a WASM module--it can only do it asynchronously. So, we import, wrap the async-ness up, and export something a little cleaner to use.